stories/navigation/Pagination.stories.js

```diff
@@ -1,35 +1,44 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
+import { addDecorator } from '@storybook/react';
 import { withInfo } from '@storybook/addon-info';
-import { action } from '@storybook/addon-actions';
 import StoryContainer from '../storyHelpers/StoryContainer';
+import { ThemeProvider } from '@sparkpost/matchbox/components/ThemeProvider';
+import { action } from '@storybook/addon-actions';
 
 import { Pagination } from '@sparkpost/matchbox';
 
-storiesOf('Navigation|Pagination', module)
-  .addDecorator((getStory) => (
-    <StoryContainer>{getStory()}</StoryContainer>
-  ))
+addDecorator(storyFn => (
+  <ThemeProvider>
+    <StoryContainer bg="white">{storyFn()}</StoryContainer>
+  </ThemeProvider>
+));
 
-  .add('with no margins', withInfo()(() => (
-    <div data-id="pagination-no-margin">
-      <Pagination
-        pages={8}
-        pageRange={3}
-        marginsHidden
-        onChange={action('Page Changed')}
-      />
-    </div>
-  )))
+export default {
+  title: 'Navigation|Pagination',
+};
+
+export const WithNoMargins = withInfo()(() => (
+  <div data-id="pagination-no-margin">
+    <Pagination mb={400} pages={8} pageRange={3} marginsHidden onChange={action('Page Changed')} />
+  </div>
+));
 
-  .add('with lots of pages and flat buttons', withInfo()(() => (
-    <div data-id="pagination-flat">
-      <Pagination
-        flat
-        pages={30}
-        pageRange={3}
-        selectedColor='navy'
-        onChange={action('Page Changed')}
-      />
+export const WithLotsOfPages = withInfo()(() => (
+  <div data-id="pagination-lots-of-pages">
```

Comment:
in `Pagination.spec.js` I think you might be able to replace `<div data-id="pagination-flat">` with `pagination-lots-of-pages` and the test will pass. Looks like the pagination props are exactly the same, just missing `flat`.

You may need to update the iframe url too since the story title changed